metron-analytics/metron-profiler-common/src/test/java/org/apache/metron/profiler/DefaultProfileBuilderTest.java

```diff
@@ -73,7 +71,7 @@ public void setup() throws Exception {
    * }
    */
   @Multiline
-  private String testInitProfile;
+  private static String testInitProfile;
```

Comment:
IntelliJ really hated it, so I changed it, because it kept me from being able to run things.  Maven will apparently deal with it, but with so much shifting underneath it, IntelliJ started to fall apart a bit at times.